cli/tests/testdata/bench/no_run.ts

```diff
@@ -0,0 +1 @@
+const _value: string = 1;
```

Comment:
Maybe add a console.log to the file to confirm that the code is not run at all, and not just that the benchmark calls themselves are not run.